src/themer.spec.tsx

```diff
@@ -0,0 +1,82 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+import 'jest';
+import * as React from 'react';
+import { mount } from 'enzyme';
+import Themer from './themer';
+import { THEME_SELECTION, HIGH_CONTRAST } from './app/constants/browserStorage';
+
+describe('themer', () => {
+    it('matches snapshot', () => {
+        localStorage.removeItem(THEME_SELECTION);
+        localStorage.removeItem(HIGH_CONTRAST);
+        const component = (
+            <Themer />
+        );
+        expect(mount(component)).toMatchSnapshot();
```

Comment:
let's use testSnapshot( ) test helper, which can help get rid of the unncessary details of the snapshot, like font, color and classnames.

Reply:
Moved to more expect calls based on state per offline conversation.